ee/clickhouse/migrations/0028_remove_dollar_from_materialized_columns.py

```diff
@@ -0,0 +1,76 @@
+from functools import lru_cache
+
+import structlog
+from infi.clickhouse_orm import migrations
+
+from ee.clickhouse.materialized_columns.columns import get_materialized_columns, materialized_column_name
+from posthog.client import sync_execute
+from posthog.constants import GROUP_TYPES_LIMIT
+from posthog.settings import CLICKHOUSE_CLUSTER, CLICKHOUSE_DATABASE
+
+logger = structlog.get_logger(__name__)
+
+
+def alias_column(table, current_name, new_name):
+    execute_on_cluster = f"ON CLUSTER '{CLICKHOUSE_CLUSTER}'" if table == "events" else ""
+    sync_execute(
+        f"""
+        ALTER TABLE {table} {execute_on_cluster}
+        ADD COLUMN IF NOT EXISTS \"{new_name}\"
+        VARCHAR ALIAS \"{current_name}\"
+    """
+    )
```

Comment:
I brought this over from the previous PR, but I'm not sure why we're using `ALIAS` instead of `COLUMN RENAME`. From what I understand, neither touch the underlying data, but using `ALIAS` means the columns don't appear in a `select *`

Reply:
After looking at the history for https://github.com/PostHog/posthog/pull/8855, it looks like this was a [very explicit decision](https://github.com/PostHog/posthog/pull/8855/commits/bd1527af1d19fc957c6ddf97c4d05320884cc53b) based on running many `RENAME`s causing issues in the past.

Reply:
Following up here, after a conversation with @fuziontech, we moved back to using `RENAME` instead of `ALIAS`. It looks we initially moved to `ALIAS` because of some cluster issues that we shouldn't be experiencing anymore, and if we do have them, we know how to fix it.

@fuziontech is planning to babysit cloud after merging this migration to ensure it goes smoothly.